### ISSUE TITLE
[6.1] jooa11y link

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#47577]
### What happened?

1.Accessibility check from an article in the admin fails

### Version

6.1

### Expected result

Article loads in popup iframe and the accessibility check is loaded

### Actual result

0 Dispatcher not set in Joomla\CMS\Plugin\CMSPlugin

|  | Function | Location |
| -- | -- | -- |
1 | () | JROOT\libraries\vendor\joomla\event\src\DispatcherAwareTrait.php:41
2 | Joomla\CMS\Plugin\CMSPlugin->traitGetDispatcher() | JROOT\libraries\src\Plugin\CMSPlugin.php:573
3 | Joomla\CMS\Plugin\CMSPlugin->getDispatcher() | JROOT\plugins\system\jooa11y\src\Extension\Jooa11y.php:109
4 | Joomla\Plugin\System\Jooa11y\Extension\Jooa11y->initJooa11y() | JROOT\libraries\vendor\joomla\event\src\Dispatcher.php:454
5 | Joomla\Event\Dispatcher->dispatch() | JROOT\libraries\vendor\joomla\application\src\AbstractApplication.php:99
6 | Joomla\Application\AbstractApplication->dispatchEvent() | JROOT\libraries\src\Application\SiteApplication.php:651
7 | Joomla\CMS\Application\SiteApplication->route() | JROOT\libraries\src\Application\SiteApplication.php:244
8 | Joomla\CMS\Application\SiteApplication->doExecute() | JROOT\libraries\src\Application\CMSApplication.php:320
9 | Joomla\CMS\Application\CMSApplication->execute() | JROOT\includes\app.php:58
10 | require_once() | JROOT\index.php:51

</div></div></div><div class="phpdebugbar" data-bodymarginbottom="true" data-theme="dark" data-openbtnposition="bottomLeft" data-hideemptytabs="false" style="box-sizing: border-box; --debugbar-background: #2a2a2a; --debugbar-background-alt: #333; --debugbar-text: #e0e0e0; --debugbar-text-muted: #aaa; --debugbar-border: #3a3a3a; --debugbar-header: #1e1e1e; --debugbar-header-text: #ccc; --debugbar-header-border: #444; --debugbar-active: #444; --debugbar-active-text: #e0e0e0; --debugbar-icons: #ccc; --debugbar-badge: #444; --debugbar-badge-text: #ccc; --debugbar-badge-active: #ccc; --debugbar-badge-active-text: #1e1e1e; --debugbar-link: #aaa; --debugbar-hover: #888; --debugbar-accent: #4f8fb3; --debugbar-accent-border: #3f7a94; --debugbar-font-sans: ui-sans-serif, system-ui, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; --debugbar-font-mono: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, &quot;Liberation Mono&quot;, &quot;Courier New&quot;, monospace; width: 1711px; font-family: ui-sans-serif, system-ui, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; background: none 0% 0% / auto repeat scroll padding-box border-box rgb(42, 42, 42); z-index: 2147483647; color: rgb(224, 224, 224); text-align: left; letter-spacing: normal; direction: ltr; border-top: 0px; font-size: 14px; line-height: 1.2em; position: fixed; bottom: 0px; left: 0px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; orphans: 2; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="phpdebugbar-resize-handle" style="box-sizing: content-box; clear: initial; text-align: left; direction: ltr; border-width: 0px 0px 1px; border-top-style: initial; border-right-style: initial; border-bottom-style: solid; border-left-style: initial; border-top-color: initial; border-right-color: initial; border-bottom-color: rgb(68, 68, 68); border-left-color: initial; border-image: initial; width: 1711px; margin: -4px 0px 0px; padding: 0px; font-weight: normal; text-decoration: none; cursor: ns-resize; background: none; height: 4px; display: block;"></div><div class="phpdebugbar-header" style="box-sizing: content-box; clear: initial; text-align: left; direction: ltr; border: 0px; width: auto; margin: 0px; padding: 0px; font-weight: normal; text-decoration: none; background-color: rgb(30, 30, 30); min-height: 32px; line-height: 16px;"><a class="phpdebugbar-restore-btn" style="box-sizing: content-box; color: rgb(204, 204, 204); text-decoration: none; clear: initial; text-align: left; direction: ltr; border: 0px; width: 16px; margin: 0px; padding: 5px 8px; font-weight: normal; float: left; font-size: 14px; height: 22px; position: relative; cursor: pointer;"></a><div class="phpdebugbar-header-left" style="box-sizing: content-box; clear: initial; text-align: left; direction: ltr; border: 0px; width: auto; margin: 0px; padding: 0px; font-weight: normal; text-decoration: none; float: left;"><a class="phpdebugbar-tab phpdebugbar-active" data-empty="false" data-collector="info" style="box-sizing: content-box; color: rgb(224, 224, 224); text-decoration: none; clear: initial; text-align: left; direction: ltr; border: 0px; width: auto; margin: 0px; padding: 5px; font-weight: normal; align-items: center; line-height: 0; display: flex; float: left; cursor: pointer; height: 22px; font-size: 13px; background: none 0% 0% / auto repeat scroll padding-box border-box rgb(68, 68, 68);"></a></div></div></div><br class="Apple-interchange-newline">
</body>
</html>
